app/views/contests/_form.html.erb

```diff
@@ -66,6 +66,13 @@
      <% end %>
    </div>
 
+   <div class="form-group">
```

Comment:
Since a single form-group will consume large margin. Do you think it is better to merge these 3 form-group to one form-group?  
Maybe still separated by \<br\> or a space.